dashboard/app/models/levels/bubble_choice.rb

```diff
@@ -47,4 +47,20 @@ def dsl_default
   def sublevels
     Level.where(name: properties['sublevels']).sort_by {|l| properties['sublevels'].index(l.name)}
   end
+
+  def summarize
+    sublevel_summary = sublevels.map do |level|
+      {
+        id: level.id,
+        title: level.display_name || level.name,
+        thumbnail_url: level.try(:thumbnail_url)
```

Comment:
TIL: `level&.thumbnail_url` allows `level` to be `nil`, but `.try(` also allows `thumbnail_url` to not even be defined on `level` 🌟